packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -443,6 +443,45 @@ class DecoratedBoxTransition extends AnimatedWidget {
   }
 }
 
+/// Animated version of an [Align] that animates it's [Align.alignment] property.
+class AlignTransition extends AnimatedWidget {
+  /// Creates an animated [Align] whose [AlignmentGeometry] animation updates
+  /// the widget.
+  ///
+  /// See also:
+  ///
+  /// * [new Align].
+  const AlignTransition({
+    Key key,
+    @required Animation<AlignmentGeometry> alignment,
+    @required this.child,
+    this.widthFactor,
+    this.heightFactor,
```

Comment:
why not make these animated too?

Reply:
though i guess it's not clear exactly how that would work...

Reply:
yeah let's not worry about animating these, and we'll address that problem when it comes around.

Reply:
SizeTransition is the dual of this transition that animates those properties.